### ISSUE TITLE
关于蛇形卷积的问题

### ISSUE THREAD

**Issue by chenzean** [YaoleiQi/DSCNet#55]
作者，你好！
         我想问一下，是不是
        x_40_0 = self.conv40(x)
        x_4x_0 = self.conv4x(x)
        x_4y_0 = self.conv4y(x)
        x_2_1 = self.conv5(torch.cat([x_40_0, x_4x_0, x_4y_0], dim=1))
这种情况才是完整的蛇形卷积呢？

**Comment by chenzean** [comment]
是不是只有上述情况才能考虑x维度和y维度？

**Comment by YaoleiQi** [comment]
是的，因为感受野的一些约束导致了这样的结果，这是当时的不得已之举，诚然，也有很多设计能够让该操作变得更优美，例如用一个可旋转的线段等等~但我们并不在此操作上做下一步工作，谢谢您的关注

**Comment by chenzean** [comment]
作者，你好！
        我发现蛇形卷积似乎在计算Flops上好像蛮大的。这个可以有进一步优化空间吗
谢谢

**Comment by haicaoha** [comment]
想问下蛇形卷积的那个偏移量Δ是需要学习的参数吗

**Comment by YaoleiQi** [comment]
偏移量Δ是需要学习的参数~
